src/components/errorPage/index.tsx

```diff
@@ -34,67 +83,62 @@ export interface Props {
 }
 
 const ErrorPage: FC<Props> = ({ statusCode, language, onButtonClick }) => {
-  const showReloadPage = statusCode === 'clientSide';
+  const PrimaryAction = config[statusCode].primaryAction;
+  const SecondaryAction = config[statusCode].secondaryAction;
+
   return (
     <TranslationProvider language={language} scopes={['errorPage']}>
       <I18nContext.Consumer>
-        {({ t }) => (
-          <PageLayout maxContentWidth="md" header={null}>
-            <Flex justifyContent="center" pt={{ base: '3xl', md: 'xl' }}>
-              <Stack align="center" spacing="4xl">
-                <SimpleGrid columns={2} spacingX="4xl">
-                  <AutoScout24AppLogo width="80px" height="51px" />
-                  <MotoScout24AppLogo width="80px" height="51px" />
-                </SimpleGrid>
-                <Divider />
-                <Stack align="center" spacing="2xl">
-                  <AspectRatio ratio={4 / 3} maxWidth="400px" width="full">
-                    <chakra.img
-                      src={illustrations[statusCode]}
-                      alt={`a ${statusCode} error occurred.`}
-                    />
-                  </AspectRatio>
-                  <Stack align="center" spacing="md">
-                    <H1 textAlign="center">
-                      {t(`errorPage.${statusCode}.title`)}
-                    </H1>
-                    <Text textAlign="center">
-                      {t(`errorPage.${statusCode}.description`)}
-                    </Text>
-                  </Stack>
-                  <SimpleGrid
-                    columns={{
-                      base: 1,
-                      sm: showReloadPage ? 2 : 1,
-                    }}
-                    alignItems="center"
-                    spacing="md"
-                  >
-                    {showReloadPage ? (
-                      <Button
-                        onClick={() => {
-                          window.location.reload();
-                        }}
-                      >
-                        {t('errorPage.reloadPage')}
-                      </Button>
-                    ) : null}
-                    <Button
-                      onClick={() => {
-                        onButtonClick?.();
+        {({ t }) => {
```

Comment:
I don't think you can consume the context like that in the component where you are defining the provider as the context won't be defined at this point (it would work for the extracted components but that will make them dependent on the parent rendering the provider)